python/example/v-clamp.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# This script is included in documentation. Adapt line numbers if touched.
+
+import arbor
+import pandas  # You may have to pip install these.
+import seaborn  # You may have to pip install these.
+
+# (1) Create a morphology with a single (cylindrical) segment of length=diameter=6 μm
+tree = arbor.segment_tree()
+tree.append(arbor.mnpos, arbor.mpoint(-3, 0, 0, 3), arbor.mpoint(3, 0, 0, 3), tag=1)
+
+# (2) Define the soma and its midpoint
+labels = arbor.label_dict({"soma": "(tag 1)", "midpoint": "(location 0 0.5)"})
+
+# (3) Create and set up a decor object
+
+decor = (
+    arbor.decor()
+    .set_property(Vm=-40)
+    .paint('"soma"', arbor.density("hh"))
+    .paint('"soma"', arbor.voltage("v_clamp/v0=-42"))
```

Comment:
must be arbor.voltage_process?

Reply:
Very true. Also means we do not auto-run this! Thanks a lot!